tests/testthat/test-rankedAbundance.R

```diff
@@ -41,7 +41,7 @@ test_that("rankedAbundance returns a data.table with the correct attributes", {
   expect_true(attr$isCutoff)
   expect_equal(names(attr$computedVariables[[1]]), c('computedVariableDetails','computedVariableMetadata'))
   expect_equal(names(attr$computedVariables[[1]]$computedVariableDetails), c('variableId','entityId','dataType','dataShape','isCollection'))
-  expect_equal(names(attr$computedVariables[[1]]$computedVariableMetadata), c('defaultRange'))
+  expect_equal(names(attr$computedVariables[[1]]$computedVariableMetadata), c('defaultRange','collectionVariable'))
```

Comment:
again think the only thing id like is for computedVariables to not be an array, and solve the problem of multiple computes per app later.